medusa/helpers/externals.py

```diff
@@ -41,8 +27,9 @@ def trakt_request(api, url):
                 app.TRAKT_ACCESS_TOKEN = api.access_token
                 app.TRAKT_REFRESH_TOKEN = api.refresh_token
                 app.instance.save_config()
-        except (TokenExpiredException, TraktException) as e:
-            logger.info(u'Could not use Trakt to enrich with externals. Cause: {cause}', cause=e)
+        except (TokenExpiredException, TraktException) as error:
+            log.info(u'Could not use Trakt to enrich with externals: {0}',
+                     error)
```

Comment:
I think consistency is important so I would generally agree, but imo it depends on the intent of the error logging.  My intent with the PR is not to interpret the original intent but to adapt it to the new style in order to finish the transition from the old logger.